src/ectrans4py/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+if(HAVE_ETRANS)
```

Comment:
I think this check is redundant because it's done in the CMakeLists.txt one level up.